platform/android/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
  - Add fallback support to local ideograph font families [#15255](https://github.com/mapbox/mapbox-gl-native/pull/15255)
 
 ### Bug fixes
+ - Fixed possible crash caused by invoking of the wrong cast layer implementation function [#15398](https://github.com/mapbox/mapbox-gl-native/pull/15398).
```

Comment:
add "a" before "possible crash"

Reply:
since the crash is only "possible", the verb clause should be "that could be caused by"

Reply:
delete "of" --> "invoking the wrong"

Reply:
the last four words are better as: "layer casting implementation function."

Reply:
layer implementation casting function - it casts `Layer::Impl` instances.

Reply:
ah yeah, that’s perfect, thanks!

Reply:
Thank you!